jung-visualization/src/main/java/edu/uci/ics/jung/visualization/BaseVisualizationModel.java

```diff
@@ -107,19 +107,21 @@ public BaseVisualizationModel(
     }
   }
 
-  private LayoutModel<N, Point2D> createLayoutModel(Network<N, E> network, Dimension layoutSize) {
-    // the default
-    SpatialSupport spatialSupport = SpatialSupport.QUAD_TREE;
-    String spatialSupportProperty = System.getProperty("layout.model");
+  private SpatialSupport getSpatialSupportPreference() {
+    String spatialSupportProperty = System.getProperty("spatial.support");
     if (spatialSupportProperty != null) {
       try {
-        spatialSupport = SpatialSupport.valueOf(spatialSupportProperty);
+        return SpatialSupport.valueOf(spatialSupportProperty);
       } catch (IllegalArgumentException ex) {
```

Comment:
if you load a web page with an invalid entry in a style sheet, does your browser crash? I think that property parsing tends to be more forgiving and to do the best it can with sensible defaults. Logging the error is helpful.

Reply:
There may be different expectations with property parsing (and we can discuss that in the relevant issue), although I don't think that your analogy is quite on: having invalid HTML won't cause my browser to crash, but it certainly can cause a page to fail to render.